cli/cmd/capture/create.go

```diff
@@ -124,10 +120,10 @@ var createCapture = &cobra.Command{
 		if nowait {
 			retinacmd.Logger.Info("Please manually delete all capture jobs")
 			if capture.Spec.OutputConfiguration.BlobUpload != nil {
-				retinacmd.Logger.Info("Please manually delete capture secret", zap.String("namespace", namespace), zap.String("secret name", *capture.Spec.OutputConfiguration.BlobUpload))
+				retinacmd.Logger.Info("Please manually delete capture secret", zap.String("namespace", *opts.Namespace), zap.String("secret name", *capture.Spec.OutputConfiguration.BlobUpload))
```

Comment:
Might be nice to DRY these zap params. Maybe make a child logger for use within this func?

Reply:
Might be nice if we didn't have a global logger

Reply:
Filed #585